spec/victor-cli/ruby_code_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe RubyCode do
+  subject { described_class.new code }
+  let(:code) { "puts 'hello'" }
+
+  it "has a Victor::SVG instance" do
+    expect(subject.svg).to be_a Victor::SVG
+  end
+
+  it "includes Victor::DSL methods" do
+    expect(subject).to respond_to :setup
+  end
+
+  describe "#evaluate" do
+    it "evaluate the ruby code" do
+      expect { subject.evaluate }.to output("hello\n").to_stdout
+    end
```

Comment:
Perhaps there should be a test here when there is a syntax error? Might be nice to make sure things like line numbers etc are still helpful.

Reply:
Right now, it shows an error without backtrace. Running with `DEBUG=1` shows the backtrace, with valid lines, but it definitely should be handler in a nicer way.

I will open an issue so one of us can take care of it post-merge.